src/js/menu/operations/TurfOperation.js

```diff
@@ -46,7 +46,7 @@ L.DNC.TurfOperation = L.DNC.Operation.extend({
         **
         */
         var eventExtras = { mapLayer: mapLayer, layerName: newLayer.name, isOverlay: true };
-        this.parent.parentDomElement.fire('operation-result', eventExtras);
+        this.parent.parent.fire('operation-result', eventExtras);
```

Comment:
Can @thebigspoon verify that this is the expected functionality?  Should `.fire` be called on the parent's parent or the parent's dom element?